src/vs/workbench/browser/parts/editor/editor.ts

```diff
@@ -41,6 +42,24 @@ export const DEFAULT_EDITOR_PART_OPTIONS: IEditorPartOptions = {
 	splitSizing: 'distribute'
 };
 
+export function computeEditorAriaLabel(input: IEditorInput, group: IEditorGroup | undefined, groupCount: number): string {
+	let ariaLabel = input.ariaLabel;
+	if (group && !group.isPinned(input)) {
+		ariaLabel += localize('preview', ", preview");
+	}
+	if (group && group.isSticky(input)) {
+		ariaLabel += localize('pinned', ", pinned");
+	}
+	// Apply group information to help identify in
+	// which group we are (only if more than one group
+	// is actually opened)
+	if (group && groupCount > 1) {
+		ariaLabel += ', ' + group.ariaLabel;
```

Comment:
I personally would prefer using template strings over `+` concatenation because it is easier to read

Reply:
Thanks, tackled now.